src/ASM/ASMs1D.C

```diff
@@ -1863,3 +1863,14 @@ Fields* ASMs1D::getProjectedFields (const Vector& coefs, size_t) const
             << std::endl;
   return nullptr;
 }
+
+
+void ASMs1D::getNeighbours (NeighArray& neigh) const
+{
+  for (int i = 0; i < static_cast<int>(nel); ++i) {
+    if (i-1 > -1)
+      neigh[i].push_back(i-1);
+    if (i+1 < static_cast<int>(nel))
+      neigh[i].push_back(i+1);
```

Comment:
For 2D and 3D you first resize and fill with -1, but here the first and last element will have only 1 element in its neighbor list. Can that be a problem? At least inconsistent.

And here the local patch-wise element index is used as `neigh` index, whereas for 2D and 3D the global one is used (from `getElmID`). So in the unlikely event of a parallel 1D multi-patch simulation this will fail, won't it?

Reply:
now consistent with the others.